app/index.html

```diff
@@ -180,6 +180,45 @@ <h5>References</h5>
             </div>
           </div>
         </div>
+        <div class="modal fade" tabindex="-1" id='download'>
+          <div class="modal-dialog modal-lg">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h5 class="modal-title">
+                    Download chemiscope
+                </h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                  <span aria-hidden="true">&times;</span>
+                </button>
+              </div>
+              <div class="modal-body">
+                <p>
+                    Chemiscope consists in a set of JavaScript modules to 
+                    visualize atomic structures and map associated data. 
+                    These modules can be assembled into all sorts of interactive
+                    visualization tools, such as that which you can use on these
+                    pages. 
+                </p>
+
+                <p>
+                    If you are a developer you may be interested in downloading
+                    the <a href="https://github.com/cosmo-epfl/chemiscope"> source code</a>,
+                    which is available on GitHub. 
+                </p>
+
+                <p>
+                    If you are a user, you may want to download a 
+                    <a href="chemiscope_standalone.html" download> 
+                    standalone version
+                    </a> of the standard viewer, which you can use to 
+                    display locally datasets prepared in the chemiscope 
+                    JSON format. 
```

Comment:
Should we add some basic instructions on how to use the standalone viewer (i.e. append the JSON file)? Or should we leave it as is and wait for the full implementation of "download current dataset as standalone"?

Reply:
I was thinking that this "static standalone viewer" can be used as a local GUI, and you have to load your dataset in it.